src/egon/data/datasets/gas_grid.py

```diff
@@ -318,25 +315,31 @@ def insert_gas_buses_abroad(scn_name="eGon2035"):
             "v_mag_pu_min",
             "v_mag_pu_max",
             "geom",
-        ]
+            "control",
+            "generator",
+            "location",
+            "unit",
+            "sub_network"
+        ],
+        errors = "ignore"
     )
-    gdf_abroad_buses["scn_name"] = "eGon2035"
-    gdf_abroad_buses["carrier"] = main_gas_carrier
+    gdf_abroad_buses["scn_name"] = scn_name
+    gdf_abroad_buses["carrier"] = gas_carrier
     gdf_abroad_buses["bus_id"] = range(new_id, new_id + len(gdf_abroad_buses))
 
-    # Add central bus in Russia
+    #Add central bus in Russia
     gdf_abroad_buses = pd.concat(
         [
             gdf_abroad_buses,
             pd.DataFrame(
-                index=[gdf_abroad_buses.index.max() + 1],
+                index=["RU"],

```

Comment:
I did not test it, but I am wondering if text entries in the index are fine or might lead to problems.

Reply:
The index of gdf_abroad_buses is now based on strings. That is why I changed it directly to "RU". At the end, this value is neglected [here](https://github.com/openego/powerd-data/blob/be1b2ff6e7157b9bf3825023f15f6668fa4399b4/src/egon/data/datasets/gas_grid.py#L348-L365), so it is only important that it creates no errors like the one you described (str + int).